cpp/tests/groupby/sort/pandas/single_column_avg.cu

```diff
@@ -0,0 +1,391 @@
+/*
```

Comment:
Referring to ignoring null keys as "Pandas" behavior doesn't convey anything to people who are unfamiliar with Pandas semantics.

Instead of calling the folder `pandas`, rename to something like `ignore_null_keys`.